index.bs

```diff
@@ -173,7 +173,7 @@ in the Generic Sensor API [[!GENERIC-SENSOR]].
 Permissions Policy integration {#permissions-policy-integration}
 ==============================
 
-This specification utilizes the [=policy-controlled feature=] identified by the string "<code><a data-lt="accelerometer-feature">accelerometer</a></code>" defined in [[DEVICE-ORIENTATION#permissions-policy-integration]].
+This specification utilizes the [=policy-controlled feature=] identified by the string "<code><a data-lt="accelerometer-feature">accelerometer</a></code>" defined in [[DEVICE-ORIENTATION#permissions-integration]].
```

Comment:
The definition of "accelerometer" is exported by [[DEVICE-ORIENTATION]] so we should be able to simplify this to,

```suggestion
This specification utilizes the [=policy-controlled feature=] identified by the string "<code><a permission>accelerometer</a></code>" defined in [[DEVICE-ORIENTATION]].
```